### DISCUSSION TITLE
Need info about header

### DISCUSSION THREAD

**Discussion by Chris2011** [Chris2011/minifierbeans#87] [Q&A] [answered]
@JuergenLaas, @DucMinh19, @Higestromm and @Ilya-Zhulin do you use the option to add header stuff to your files?

![image](https://user-images.githubusercontent.com/795658/130122025-3730bb53-aed8-4c49-bf55-4762e1979cae.png)

If yes, what do you add to it, please make some screenshots or please send some snippets. Thx for your help.

**Comment by JuergenLaas** [comment]
Thanks for asking, no I don't use header in my files. Before, I used this line "@charset-UTF8"; in my CSS files. Now I don't.

**Comment by DucMinh19** [comment]
I don't also use header. Now I only set  staticly the minify options in source since can not store them with Netbean (win 10 )
![Capture](https://user-images.githubusercontent.com/88248041/130323282-8395fb0c-dd27-4ec1-b4b8-2aa8e1db4689.PNG)

**Comment by Chris2011** [comment]
I'm on it. Still work in progress: #73. Will be ready in a couple of days. This thread here is not about that topic.

**Comment by Ilya-Zhulin** [comment]
Hi.. If I get it correctly, I use it to add something like "/* Created by me in 2021 */"... It is not principally, but sometimes it's useful for community work.

**Comment by Chris2011** [comment, marked as answer]
Ok, maybe for now, I will hide this option and will add it later. The CLIs that I use underneath (Node based) don't have any CLI option to add extra content. So I need to wait for the output file, read it, prepend extra content to it and save it again. So for now I'm fixing #73 as mentioned and will add this feature later.

**Comment by Chris2011** [comment]
Thx for all the information.
